watchfs/watchfs_test.go

```diff
@@ -10,6 +10,8 @@ import (
 
 	"github.com/google/go-cmp/cmp"
 	"github.com/qri-io/qri/event"
+	reporef "github.com/qri-io/qri/repo/ref"
+	repotest "github.com/qri-io/qri/repo/test"
```

Comment:
a bit wary of how much this relies on repo, but until dscache can reliable give us a list of references, this is what we got!